crates/nu-data/src/value.rs

```diff
@@ -142,6 +149,13 @@ pub fn compute_values(
                         Ok(UntaggedValue::Primitive(Primitive::Int(x % y)))
                     }
                 }
+                Operator::Pow => {
+                    // only get the first part of the u32 digits as the power multiplier
+                    let bigint_as_u32 = y.to_u32_digits();
+                    Ok(UntaggedValue::Primitive(Primitive::Int(
+                        x.pow(bigint_as_u32.1[0]),
```

Comment:
operations on bigint and bigdecimal shouldn't leave the big form and go back to floats and ints.

Reply:
I changed this to use `ToPrimitive::to_u32` and I check the option. Hopefully no one tries to `pow` with something bigger than a `u32`.